extensions/binary_glTF/README.md

```diff
@@ -171,12 +184,17 @@ Using the 1200 12th Ave model (thanks to [Cube Cities](http://cubecities.com/)),
 
 * Cesium COLLADA-to-glTF Converter ([app](http://cesiumjs.org/convertmodel.html))
 * colladaToBglTFConverter ([code](https://github.com/virtualcitySYSTEMS/colladaToBglTFConverter))
+* SRC writer source code ([code](http://x3dom.org/src/files/src_writer_source.zip))
 
 ## Resources
 
 * Discussion - [#357](https://github.com/KhronosGroup/glTF/issues/357)
 * base64-encoded data in glTF - [#68](https://github.com/KhronosGroup/glTF/issues/68)
 * [Faster 3D Models with Binary glTF](http://cesiumjs.org/2015/06/01/Binary-glTF/) article on the Cesium blog
+* SRC project page (paper, background, basic writer) - [http://x3dom.org/src/](http://x3dom.org/src/)
 
 <a name="BenchData">
 * [1] Raw data for benchmarks using compression available in [BenchData](BenchData/README.md) supplemental.
```

Comment:
Can you tweak `BenchData/README.md` to say that the tests were done with `CESIUM_binary_glTF`, but the difference is insignificant?